main.tf

```diff
@@ -30,5 +32,5 @@ module "tamr_instance" {
   volume_size              = var.volume_size
   enable_volume_encryption = var.enable_volume_encryption
   bootstrap_scripts        = var.bootstrap_scripts
-  additional_tags          = var.tamr_instance_tags
+  tags                     = merge(var.tags, var.tamr_instance_tags)
```

Comment:
With the default of tamr_instance tags being:
```
variable "tamr_instance_tags" {
  type        = map(string)
  description = "Additional tags to be attached to the Tamr EC2 instance"
  default     = { Author : "Tamr", Name : "Tamr VM" }
}
```

Which variable gets precedent in this merge operation?

We're using `Author` in all of our examples in var.tags, and I'm worried that it wouldn't be used in this case. The quick test would be to set author to something other than "Tamr" in the example to see if it's used.

We may want to update `tamr_instance_tags` to not include author, or maybe go further and make the name tag a special case tag that will be added by the module, since that's a common tag that someone would want to set.

Reply:
Hello @souza-dan and thanks for your response. The variable getting precedence with the merge is the one at the very right, in this case `tamr_instance_tags`. In this case if the customer opts to use `Auhor: Tamr` as `var.tags` and `Author: Customer` as `tamr_instance_tags` the `Author: CustomerX` will take precedence.

Reply:
Thanks, I should have just read the doc!
https://www.terraform.io/docs/language/functions/merge.html